test/__snapshots__/StatsTestCases.test.js.snap

```diff
@@ -1594,102 +1594,102 @@ You may need an appropriate loader to handle this file type.
 `;
 
 exports[`StatsTestCases should print correct stats for performance-different-mode-and-target 1`] = `
-"Hash: b27a7019f90a13ead012bb84b396eb1e98365d94fb930f6aa3329ab03a278deb0030bfebe86abb37a1cddcae25eb52f3112b62c8b61bc84829d26b13ae06efe5f0099a11af36
+"Hash: c5d2dc589972a83203dd11fba64cb47a40a00e5cfdbbddc0be94fefb21a336f975e87fc99cd9884e1834585938a82ed7187bc9627ea3b773a833ffd632e42d4909027b03dc2a
 Child
-    Hash: b27a7019f90a13ead012
+    Hash: c5d2dc589972a83203dd
     Time: Xms
     Built at: Thu Jan 01 1970 00:00:00 GMT
                  Asset     Size  Chunks                    Chunk Names
-    warning.pro-web.js  297 KiB       0  [emitted]  [big]  main
+    warning.pro-web.js  735 KiB       0  [emitted]  [big]  main
```

Comment:
This change is strange. Why is there such a big difference?

Reply:
I changed the test fixtures. Previously, they were 400kb files containing a repeated single character, but that gzips to a very tiny output. In order to make the tests work and add some realism, I changed them to be generated JSON structures.

Reply:
Thanks!